### PR TITLE
Add basic documentation

### PR DESCRIPTION
**Pull request by mkaito**
It's mostly boiler plate with a link to HexDocs, but it's better than nothing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# ConfigMacro
+
+[![Release](https://img.shields.io/github/release/serokell/patak.svg)](https://github.com/serokell/patak/releases)
```
Comment (gromakovsky) [inline README.md]: Should be removed.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+ConfigMacro contains `config/2` macro that defines module-namespaced `Application.get_env/3` getters.
+Supposed to complement `Mix.Config.config/3` macro.
+
+# Documentation
+
+See usage documentation on [HexDocs](https://hexdocs.pm/config_macro/ConfigMacro.html).
+
+## For Contributors [↑](#-patak)
```
Comment (gromakovsky) [inline README.md]: Please remove these `(#-patak)` or turn them into working links.

**Review by gromakovsky — CHANGES_REQUESTED** [review]
There are some leftovers from the template.

**Comment by mkaito** [comment]
Thanks, I missed those. Should be fixed now.

**Review by gromakovsky — APPROVED** [review]

**Pull request merged**